docs/concepts/pdp/configuration.mdx

```diff
@@ -315,6 +329,63 @@ Default: `False`
 
 Enable FactDB for this PDP. For more information, see the [FactDB documentation](/concepts/pdp/factdb).
 
+#### PDP_CACHE_STORE
+
+Default: `none`
+
+The cache storage type. Options include: none, in-memory, and redis.
+For more information, see the [PDP Cache documentation](/concepts/pdp/overview#pdp-caching-mechanism).
+
+_Added in PDP v0.9.0_
+
+#### PDP_CACHE_TTL
+
+Default: `3600`
+
+Time-to-live in seconds for cached items.
+
+_Added in PDP v0.9.0_
+
+#### PDP_CACHE_MEMORY_CAPACITY
+
+Default: `128`
+
+Memory capacity in MiB for in-memory cache (when using in-memory store).
+
+_Added in PDP v0.9.0_
+
+#### PDP_CACHE_REDIS_URL
+
+Default: "" (empty string)
+
+Redis connection URL (when using redis store).
+
+_Added in PDP v0.9.0_
+
+#### PDP_HORIZON_PORT
+
+Default: `7001`
+
+The port for the Horizon service.
```

Comment:
Should explain what is the horizon service , at least briefly (the API interface for the PDPs own services)